server/src/main/java/io/druid/server/coordinator/DruidCoordinatorConfig.java

```diff
@@ -49,6 +49,16 @@ public boolean isConvertSegments()
     return false;
   }
 
+  @Config("druid.coordinator.kill.on")
+  public boolean isKillSegments()
```

Comment:
Is this something we want to be able to change without resetting the coordinator?